nextflow.config

```diff
@@ -185,16 +207,14 @@ profiles {
     test_fetchimgt { includeConfig 'conf/test_fetchimgt.config' }
 }
 
+
 // Load igenomes.config if required
 if (!params.igenomes_ignore) {
     includeConfig 'conf/igenomes.config'
 } else {
     params.genomes = [:]
 }
 
-// Load modules.config for DSL2 module specific options
-includeConfig 'conf/modules.config'
```

Comment:
Is there a reason to do this? You still have MultiQC and FastQC for example (or are they not used, and just left here for template-merge reasons?)

Reply:
the import is just moved to some lines below, to have it at the same spot as the template

Reply:
Ahh missed that, :+1: